test/e2e/service.go

```diff
@@ -808,7 +808,7 @@ var _ = Describe("Services", func() {
 })
 
 func waitForLoadBalancerIngress(c *client.Client, serviceName, namespace string) (*api.Service, error) {
-	const timeout = 4 * time.Minute
+	const timeout = 20 * time.Minute
```

Comment:
nit: Please add comment explaining the ridiculously long timeout, with internal reference to b/21807001